reference/5.1/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md

```diff
@@ -548,20 +523,20 @@ parameter should contain **one** at **index 0** and **two** at **index 1**.
 Instead, both elements are combined into a single entity.
 
 ```powershell
-function Test-Remainder
-{
-     param(
-         [string]
-         [Parameter(Mandatory, Position=0)]
-         $Value,
-         [string[]]
-         [Parameter(Position=1, ValueFromRemainingArguments)]
-         $Remaining)
-     "Found $($Remaining.Count) elements"
-     for ($i = 0; $i -lt $Remaining.Count; $i++)
-     {
+function Test-Remainder {
+    param(
+        [Parameter(Mandatory, Position=0)]
+        [string]$Value,
+
+        [Parameter(Position=1, ValueFromRemainingArguments)]
+        [string[]]$Remaining
+    )
+
+    "Found $($Remaining.Count) elements"
+    for ($i = 0; $i -lt $Remaining.Count; $i++)
+    {
```

Comment:
```suggestion

    for ($i = 0; $i -lt $Remaining.Count; $i++) {
```

Should we use our usual brace style here, and separate the prior output from the loop?